src/components/CardsBox.jsx

```diff
@@ -0,0 +1,14 @@
+import Card from "./Card";
+import "./CardsBox.css";
+const CardsBox = ({ folderUser }) => {
+  const linkDatas = folderUser.folder?.links;
+  if (!linkDatas) return;
+  return (
+    <div className='CardsBox'>
+      {linkDatas.map((data) => {
+        return <Card key={data.id} data={data} />;
+      })}
```

Comment:
한 줄로 리턴할 때는 return 을 안써주는 게 더 가독성이 좋아요! 특히 렌더부에서요!
```suggestion
      {linkDatas.map((data) => <Card key={data.id} data={data} />)}
```